src/airflow/dags/airqo_measurements.py

```diff
@@ -483,8 +483,62 @@ def update_latest_data_topic(data: pd.DataFrame):
     update_latest_data_topic(calibrated_data)
 
 
+@dag(
+    "AirQo-Raw-Data-Low-Cost-Measurements",
+    schedule="*/5 * * * *",
+    default_args=AirflowUtils.dag_default_configs(),
+    catchup=False,
+    tags=["airqo", "raw", "low cost"],
+)
+def airqo_raw_data_measurements():
+    import pandas as pd
+
+    @task(
+        doc_md=extract_raw_airqo_data_doc,
+    )
+    def extract_raw_data():
+        from airqo_etl_utils.airqo_utils import AirQoDataUtils
+        from airqo_etl_utils.constants import DeviceCategory
+        from airqo_etl_utils.date import date_to_str_hours
+        from datetime import datetime, timedelta
+
+        hour_of_day = datetime.utcnow() - timedelta(minutes=30)
+        start_date_time = date_to_str_hours(hour_of_day)
+        end_date_time = datetime.strftime(hour_of_day, "%Y-%m-%dT%H:59:59Z")
+
+        return AirQoDataUtils.extract_devices_data(
+            start_date_time=start_date_time,
+            end_date_time=end_date_time,
+            device_category=DeviceCategory.LOW_COST,
+        )
+
+    @task(
+        doc_md=clean_data_raw_data_doc
+    )
+    def clean_data_raw_data(data: pd.DataFrame):
+        from airqo_etl_utils.airqo_utils import AirQoDataUtils
+
+        return AirQoDataUtils.clean_low_cost_sensor_data(data=data)
+
+    @task(
+       doc_md= send_raw_measurements_to_bigquery_doc
```

Comment:
Hi @BenjaminSsempala 
How is `send_raw_measurements_to_bigquery_doc` imported into this file?

Reply:
Thanks @Noah, I've updated it